src/components/shapes/roundedRect.tsx

```diff
@@ -19,44 +19,72 @@ class RoundedRectGeometry extends THREE.ExtrudeGeometry {
     shape.lineTo(-width / 2, height / 2 - radius);
     shape.quadraticCurveTo(-width / 2, height / 2, -width / 2 + radius, height / 2);
 
-    const extrudeSettings: THREE.ExtrudeGeometryOptions = {
+    const extrudeSettings: ExtrudeGeometryOptions = {
       depth,
-      bevelEnabled: true,
-      bevelThickness: 0.11,
+      bevelEnabled: false,
       curveSegments: 6,
+      UVGenerator: {
```

Comment:
I have no idea what this is doing😂 Can you explain and maybe add a comment?

Reply:
Sure =)) It's a mess needed to ensure 3 faces: back,front,sides. Otherwise it's all messed up.

I'll add comments